gatsby-config.js

```diff
@@ -6,6 +6,7 @@ require('dotenv').config({
 
 module.exports = {
   siteMetadata: {
+    siteUrl: 'https://www.rebuildblackbusiness.com/',
```

Comment:
@molebox duplicate key here check line 33

Reply:
Ahhh didnt see that! :-(